plugin-hrm-form/src/permissions/fetchRules.ts

```diff
@@ -14,21 +14,20 @@
  * along with this program.  If not, see https://www.gnu.org/licenses/.
  */
 
-const zmRules = require('./zm.json');
+import type { RulesFile } from '.';
+import { fetchPermissionRules } from '../services/PermissionsService';
+
+const closedRules = require('./closed.json');
+const e2eRules = require('./e2e.json');
 
 // TODO: do this once, on initialization, then consume from the global state.
-export const fetchRules = (permissionConfig: string) => {
+export const fetchRules = async (permissionConfig): Promise<RulesFile> => {
   try {
-    // eslint-disable-next-line global-require
-    const rules = require(`./${permissionConfig}.json`);
-
-    if (!rules) throw new Error(`Cannot find rules for ${permissionConfig}`);
+    if (permissionConfig === 'e2e') return e2eRules;
 
-    return rules;
+    return await fetchPermissionRules();
   } catch (err) {
-    const errorMessage = err.message ?? err;
-    console.error('Error fetching rules, using fallback rules. ', errorMessage);
-
-    return zmRules;
+    console.error('Error fetching rules:', err);
+    return closedRules;
```

Comment:
```suggestion
    return null;
```

This along with the suggestion below should eliminate the need for a 'closed.json' ruleset that will need maintaining going forward (it might break tests though)